pkg/dependency/pypi.go

```diff
@@ -115,11 +115,17 @@ func (p PyPi) getReleases() ([]DepVersion, error) {
 				return nil, fmt.Errorf("could not find sha256 for version %s", version)
 			}
 
+			var cpe string
+			if p.productName == "pip" {
+				cpe = fmt.Sprintf("cpe:2.3:a:pypa:pip:%s:*:*:*:*:python:*:*", version)
```

Comment:
Why are we limiting this to python rather than using a `*`?

Reply:
We weren't sure what alternatives there were to python being there but we assumed that if there were any we wouldn't want them. Doing a search for `cpe:2.3:a:pypa` only shows python though, so a `*` is probably fine here.

Reply:
Seems like `python` is more correct in case they do add alternatives so I think it's ok to leave it as is.

One other question, though- seems like you weren't able to find a CPE for `pipenv` and I wasn't either so are we planning to just not include a CPE for that one?

Reply:
Yeah it didn't seem like pipenv was in this database at all so it didn't seem right to include it.

Reply:
👍  makes sense.